### PR TITLE
Adds Luna, Sevastopol, UN decals; Removes Matchmaking; Causes Bxil's brain to melt down

### PR DESCRIPTION
**Pull request by ExpanseStation**
Going to break a lot o things; we need to get rid of some stuff.

removes laws, matchmaking, adds luna and the sevastpool.

### CODE REVIEW

File: code/modules/power/smes.dm [hunk 1]
```diff
@@ -348,10 +348,9 @@
 	return 1
 
 /obj/machinery/power/smes/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1)
-
```
Comment (Bxil) [inline code/modules/power/smes.dm:351]: Mhmmm, random loss of lines in random files.

File: maps/luna/luna_unit_testing.dm [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/datum/map/example
```
Comment (Bxil) [inline maps/luna/luna_unit_testing.dm]: What is this?

File: maps/sevastopol/sevastopol_define.dm [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+	faction_choices = list(
+		"UN",
+		"MCR",
+		"OPA",
+		"Voltaire Collective"
+		)
+
+	religion_choices = list(
+		"Jew",
```
Comment (Bxil) [inline maps/sevastopol/sevastopol_define.dm]: Jewish

File: maps/sevastopol/sevastopol_define.dm [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		"Hindu",
+		"Buddhist",
+		"Muslim",
+		"Christian",
+		"Agnostic",
+		"Deist",
+		"Athiest",
```
Comment (Bxil) [inline maps/sevastopol/sevastopol_define.dm]: Atheist

File: maps/sevastopol/sevastopol_unit_testing.dm [hunk 5]
```diff
@@ -0,0 +1,6 @@
+/datum/map/example
```
Comment (Bxil) [inline maps/sevastopol/sevastopol_unit_testing.dm]: What this be again?

**Review by Bxil — COMMENTED** [review]
Also, normally I would scream about breaking the Torch completely, but we are going to remove it anyways I take it.

File: maps/sevastopol/sevastopol_define.dm (hunk 3 above)
Comment (ExpanseStation) [reply maps/sevastopol/sevastopol_define.dm]: that was @rickmund

File: maps/sevastopol/sevastopol_define.dm (hunk 4 above)
Comment (ExpanseStation) [reply maps/sevastopol/sevastopol_define.dm]: @rickmund fine work again

File: code/modules/power/smes.dm (hunk 1 above)
Comment (ExpanseStation) [reply code/modules/power/smes.dm:351]: that would be @rickmund

File: maps/luna/luna_unit_testing.dm (hunk 2 above)
Comment (Bxil) [inline maps/luna/luna_unit_testing.dm]: Actually this should be `/datum/map/luna`

File: maps/sevastopol/sevastopol_unit_testing.dm (hunk 5 above)
Comment (Bxil) [inline maps/sevastopol/sevastopol_unit_testing.dm]: `/datum/map/sevastopol`

**Review by Bxil — CHANGES_REQUESTED** [review]

**Pull request closed without merging**